source/faq/questions/dashboard-faq.md

```diff
@@ -82,6 +82,12 @@ We have already begun the implementation for capturing even more things from you
 
 These will be added in subsequent releases.
 
+## {% fa fa-angle-right %} Why are my video recordings freezing or dropping frames when running in CI?
+
+Videos recorded to the Dashboard may freeze up or drop frames if there are not enough resources available when running the tests in your CI container. Like with any application, there needs to be the required CPU to run Cypress and record video. You can run your tests with {% url 'memory and CPU logs enabled' troubleshooting#Log-memory-and-CPU-usage %} to identify and evaluate the resource utilization within your CI.
```

Comment:
I think let's not say this is the "Dashboard" problem - because this is CI problem. Even if they store the videos as test artifacts without using the Dashboard, it will still be the same. Probably we can say "Videos recorded on Continuous Integration servers may have frozen or dropped frames..."

Reply:
Good point. Should I move to Using-Cypress-FAQ instead, if it's not related to Dashboard?

Reply:
I think it makes sense there, good suggestion